target/linux/ramips/image/Makefile

```diff
@@ -56,6 +57,22 @@ define Build/loader-kernel
 	$(call Build/loader-common,LOADER_DATA="$@")
 endef
 
+define Build/loader-common-sercomm
+	rm -rf $@.src
+	$(MAKE) -C lzma-loader \
+		PKG_BUILD_DIR="$@.src" \
+		TARGET_DIR="$(dir $@)" LOADER_NAME="$(notdir $@)" \
+		BOARD="$(BOARDNAME)" PLATFORM="$(LOADER_PLATFORM)" \
+		LZMA_TEXT_START=0x82800000 LOADADDR=0x80001000 \
```

Comment:
So, the only difference to the existing recipe is the TEXT_START value? This should be made a variable instead of copying the whole recipe (don't we already have a solution for that?) ..

Reply:
I have done this. It works, but looks a little bit strange. I'm ready to improve this, but I need a piece of advice.  

P.S. When I added KERNEL_LOADADDR := 0x81001000 (and LZMA_TEXT_SIZE too) to the this device definition in mt7621.mk then it has broken another devices (tested on Xiaomi RM2100). KERNEL_LOADADDR := 0x81001000 applied to the Xiaomi too. Currently I don't understand why.

Reply:
This conversation is no longer relevant due to code changes.